tools/sanity_check/local_sanity_check.py.in

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python@PY_VERSION@
+
+# Copyright (C) 2012 Red Hat, Inc.
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
```

Comment:
Since this is a utility and not a library, we should probably pick a non-library license?  Not sure if we did this with other utilities.

Reply:
Copied many code from `lsmcli` which LGPL. If we re-license those lines to BSD or MIT or other license, I need permission from these authors(I haven't checked which line belong to who yet):
```bash
$ git log --pretty=format:"%an%x09" tools/lsmcli/cmdline.py | sort -u
Gris Ge
Joe
Joe Handzik
Ma Shimiao
Tony Asleson
```

So I guess we stick with LGPL. :)